webpack/JobInvocationDetail/JobInvocationConstants.js

```diff
@@ -29,3 +34,63 @@ export const DATE_OPTIONS = {
   hour12: false,
   timeZoneName: 'short',
 };
+
+const getColumnsStatus = ({ hostJobStatus, jobStatus }) => {
+  const defaultTitle = jobStatus === 'queued' ? __('Scheduled') : __('Unknown');
+
+  switch (hostJobStatus) {
+    case 'success':
+      return { title: __('Succeeded'), status: 0 };
+    case 'error':
+      return { title: __('Failed'), status: 1 };
+    case 'planned':
+      return { title: __('Scheduled'), status: 2 };
+    case 'running':
+      return { title: __('Pending'), status: 3 };
+    case 'cancelled':
+      return { title: __('Cancelled'), status: 4 };
+    default:
+      return { title: defaultTitle, status: 5 };
+  }
+};
+
+export const columns = {
+  name: {
+    title: __('Name'),
+    wrapper: ({ name }) => <a href={`/new/hosts/${name}`}>{name}</a>,
+    weight: 1,
+  },
+  groups: {
+    title: __('Host group'),
+    wrapper: ({ hostgroup_id, hostgroup_name }) => (
+      <a href={`/hostgroups/${hostgroup_id}/edit`}>{hostgroup_name}</a>
+    ),
+    weight: 2,
+  },
+  os: {
+    title: __('OS'),
+    wrapper: ({ operatingsystem_id, operatingsystem_name }) => (
+      <a href={`/operatingsystems/${operatingsystem_id}/edit`}>
+        {operatingsystem_name}
+      </a>
+    ),
+    weight: 3,
+  },
+  smart_proxy: {
+    title: __('Smart proxy'),
+    wrapper: ({ smart_proxy_name, smart_proxy_id }) => (
+      <a href={`/smart_proxies/${smart_proxy_id}`}>{smart_proxy_name}</a>
```

Comment:
Same here, if the values are empty they shouldnt be links (Not sure its possible for os, but still)